lstchain/io/lstcontainers.py

```diff
@@ -266,3 +267,43 @@ class DL1MonitoringEventIndexContainer(Container):
     calibration_id = Field(-1, 'Index of calibration event for DL1 file')
     pedestal_id = Field(-1, 'Index of pedestal event for DL1 file')
     flatfield_id = Field(-1, 'Index of flat-field event for DL1 file')
+
+class LSTEventType:
+    """
+    Class to recognize event type from trigger bits
+    bit 0: Mono
+    bit 1: stereo
+    bit 2: Calibration
+    bit 3: Single Phe
+    bit 4: Softrig(from the UCTS)
+    bit 5: Pedestal
+    bit 6: slow control
+    bit 7: busy
+    """
+
+    def is_mono(trigger_type):
```

Comment:
These can all be staticmethods.

I see that this makes sense to have in a common namespace, but this class is not a valid python class (even if it seems to work for you at the moment). 

The first argument to any normal method should be `self`, the reference to the instance, which is automatically inserted.

If you have free functions (that don't refer to any data in the class, nor to the class itself, use `@staticmethod`
So at least add `@staticmethod` above each method here.